<launch>

    <node name="tracking_finite_time_asmc" pkg="sensors" type="tracking_finite_time_asmc" >
    <param name = "k_x" value = "0.02" />
	<param name = "k_y" value = "0.02" />
	<param name = "kmin_x" value = "0.001" />
	<param name = "kmin_y" value = "0.001" />
	<param name = "k2_x" value = "0.001" />
    <param name = "k2_y" value = "0.001" />
	<param name = "mu_x" value = "0.1" />
	<param name = "mu_y" value = "0.1" />
    <param name = "lambda_x" value = "0.001" />
	<param name = "lambda_y" value = "0.001" />
    <param name = "L1_xy" value = "0.15" />
	<param name = "tc_xy" value = "10" />

    <param name = "k_u" value = "0.2" />
	<param name = "k_r" value = "0.4" />
	<param name = "kmin_u" value = "0.01" />
	<param name = "kmin_r" value = "0.01" />
	<param name = "k2_u" value = "0.01" />
    <param name = "k2_r" value = "0.01" />
	<param name = "mu_u" value = "0.03" />
	<param name = "mu_r" value = "0.03" />
    <param name = "lambda_u" value = "0.001" />
	<param name = "lambda_r" value = "0.001" />
    <param name = "L1_u" value = "0.15" />
    <param name = "L1_r" value = "0.5" />
	<param name = "tc_v" value = "3" />
    </node>

</launch>
